lib/facil/http/parsers/http1_parser.c

```diff
@@ -445,13 +445,13 @@ size_t http1_fio_parser_fn(struct http1_fio_parser_args_s *args) {
                                               : args->on_request)(args->parser))
       goto error;
     args->parser->state =
-        (struct http1_parser_protected_read_only_state_s){0, 0, 0};
+        (struct http1_parser_protected_read_only_state_s){0, 0, 0, 0};
```

Comment:
Unless compiling C++ code, this isn't necessary. Unspecified fields are automatically initialized to zero if missing (which is also true for the 2 zeros I wrote after the first zero).

Reply:
I just noticed that on a high enough warning level, you get the `missing field` warning (turned into an error). This is an informative warning, but it's probably worth avoiding.

Reply:
I don’t think ignoring the warnings is such good idea....

Reply:
You're right, it's worth avoiding the warning.